src/main/java/net/sf/jabref/logic/importer/fileformat/BibtexParser.java

```diff
@@ -119,6 +124,16 @@ public static ParserResult parse(Reader in, ImportFormatPreferences importFormat
         return Optional.of(entries.iterator().next());
     }
 
+    @Override
+    public List<BibEntry> parseEntries(InputStream inputStream) throws ParserException {
+        BufferedReader reader = new BufferedReader(new InputStreamReader(inputStream, StandardCharsets.UTF_8));
```

Comment:
At least in the tests it's useful to have it fixed to UTF8. But we should be able to support other encodings for import.